CONTRIBUTING.md

```diff
@@ -287,6 +287,66 @@ If the SVG is sourced from:
 
 In general, make sure the URL does not contain any tracking identifiers.
 
+#### Aliases
```

Comment:
Our contributing guidelines are starting to get a bit cluttered to the point where new contributors are starting to be confused by them. As aliases are very much secondary information, should we document them in a separate file linked from `CONTRIBUTING.md`? Alternatively we could, separately to this PR, look into rewriting the guidelines completely to provide better clarity for new contributors.

Reply:
I'd be in favor of keeping one document, this keeps it easily searchable with <kbd>Ctrl</kbd>+<kbd>F</kbd>.

I'm fine with restructuring but as I said before I think that will create a lot of discussion so would prefer to do that seperately.